cocos/tween/actions/action-interval.ts

```diff
@@ -389,6 +394,19 @@ export class Sequence extends ActionInterval {
         action._reversed = true;
         return action;
     }
+
+    updateWorkerTarget<T> (workerTarget: T): void {
```

Comment:
Why need this function?

Reply:
It's used for `update` the workerTarget (subTarget or call it runningTarget) in Sequence/Spawn actions.
Sequence and Spawn actions are containers for storing some other actions ( child actions) .  Child actions may have different target to apply to, so we need a method to `update` the target.